lib/components/LinearProgressBar/LinearProgressBar.scss

```
@@ -0,0 +1,108 @@
+$inactive_color: #999999;
+$success_color: #4bb4b3;
+$warning_color: #ffad3a;
+$error_color: #ec3f3f;
+$active_color:#d6dadb;
+$border_section:#f2f3f4;
+$state_hover:#e6e9e9;
+$BORDER_RADIUS:4px;
+
+.deriv-progress-slider {
```

```suggestion
.deriv-linear-progress-bar{
```
just to align with the components name maybe? 🤔